### PR TITLE
Vid columns

### LINKED ISSUE #285: column variant with video embeds
https://main--shredit--stericycle.aem.page/en-us/about/company-overview/who-we-are

https://main--shredit--stericycle.aem.live/en-us/professional-shredders-shred-it-and-skateboarding-legend-tony-hawk-team-up	
Screenshot 2024-10-04 at 11 42 21 AM

### PR DESCRIPTION
**Pull request by MarquiseRosier**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #285

Test URLs:
- Before: 
  - https://main--shredit--stericycle.aem.page/en-us/about/company-overview/who-we-are
- After: 
  - https://vid-columns--shredit--stericycle.aem.page/en-us/about/company-overview/who-we-are

### CODE REVIEW

File: blocks/columns/columns.js [hunk 1]
```diff
@@ -34,6 +34,31 @@ export function applySplitPercentages(block) {
   }
 }
 
+function embedWistia(url) {
```
Comment (jindaliiita) [inline blocks/columns/columns.js:37]: can we somehow merge these ?
https://github.com/aemsites/stericycle-shared/blob/59cef1134de57d53139485d79d5b26f1dcb417fb/blocks/video/video.js#L54

File: scripts/scripts.js [hunk 2]
```diff
@@ -158,6 +158,26 @@ function consolidateOfferBoxes(main) {
   }
 }
 
+/**
+ * get embed code for Wistia videos
+ *
+ * @param {*} url
+ * @returns
+ */
+export function embedWistia(url) {
```
Comment (jindaliiita) [inline scripts/scripts.js:167]: what about replacePlaceholder, autoplay options

Comment (MarquiseRosier) [reply scripts/scripts.js:167]: Hi, @jindaliiita can we approve this to get these blocks done?  This also does not need autoplay and placeholder, this is why I did not include them.

Comment (MarquiseRosier) [reply scripts/scripts.js:167]: If these are needed; the blocks can always be updated by the team that takes over; but all of the Wistia embeds on the site; don't have placeholders, or autoplay option.

**Review by jindaliiita — APPROVED** [review]

**Pull request merged**